### DISCUSSION TITLE
How can I adjust the source wavelet by myself in Devito

### DISCUSSION THREAD

**Discussion by acse-yw11823** [devitocodes/devito#2428] [General]
Hello, I am currently comparing results between Spyro and Devito and have noticed a slight difference in the source Ricker wavelets between them, possibly due to different settings for delay times. Is there a way I can define my own source wavelet to ensure consistency?

Many thank!!

**Comment by mloubout** [comment]
Yes you can define your own source. either as a subclass of the existing one or create a new one that implement the wavelet you are interested in.
